src/kOS/Module/kOSCustomParameters.cs

```diff
@@ -104,6 +104,11 @@ public int InstructionsPerUpdate
                                          "button on it instead of both it and the stock toolbar.")]
         public bool useBlizzyToolbarOnly = false;
 
+        [GameParameters.CustomFloatParameterUI("Default terminal brightness", minValue = 0f, maxValue = 1f, stepCount = 50, displayFormat = "N2",
+                                         toolTip = "Brightness of a kOS terminal when it appears for the first time in a scene.\n" +
+                                         "(You must reload the scene to see the effect of any changes to this slider.)")]
```

Comment:
I also considered making the GUI widget here more UI friendly by making it go from 0% to 100% rather than from 0.0 to 1.0, but then it would have different semantics from the script's settings and I figured making them agree is less bad than making the GUI slider look weird because it goes from 0 to 1.